### DISCUSSION TITLE
Select Starting Weapons

### DISCUSSION THREAD

**Discussion by Quatre112** [IntelOrca/biorand#686] [Ideas]
It would be possible to make it so you could specifically select starting weapons and maybe some starting items I'm guessing and I think that would be beneficial for people who dont like complete weapon randomization and want to have a bit of staticness in a run when it comes to what weapons they are using.

For me in RE3 I really like using the assault rifle but so far I dont start with it often because I dont know what its classified as, is it primary or powerful? Even then I'd like to always start with it and maybe the western custom because I like its firing animation. Just a thought. Noobs would appreciate it too.

**Comment by IntelOrca** [comment]
You can use the debug menu to give yourself exactly what you want at the start. The problem with adding more customisation is that it requires a longer seed which is already getting too long and may soon no longer fit in the QR.
